deploy/gke/troubleshooting.md

```diff
@@ -14,11 +14,13 @@ to identify it as the Workload Identity account.
 If you don't already have one, create a GCP service account to authorize the collector to send metrics, traces, and logs:
 
 ```
-export GCLOUD_PROJECT=<your GCP project>
+export GCLOUD_PROJECT=<the project ID of the Google Cloud project of your IAM service account>
```

Comment:
`the project ID of the Google Cloud project of your IAM service account`  maybe rephrase to be clearer?

Reply:
The current one is taken directly from the GKE documentation but how about - 
```
The Google cloud project ID in which your IAM service account resides 
```
I don't have a strong feeling on this, so open to suggestions here.